### ISSUE TITLE
Step 4

### ISSUE THREAD

**Issue by J12345654321** [jjjchens235/bing-rewards#27]
I’m new at this and need some help if possible.  Can somebody ELI5 step 4?

**Comment by jjjchens235** [comment]
This is the run setup.py step correct?

You need to run the code within the setup.py file. In your command line application (either terminal for Mac, or command prompt for windows), you need to enter the following:

1. cd
2. python /Users/jwong/Programming/Python/bing-rewards-master/setup.py (you will need to alter this slightly based on the SPECIFIC PATH of your setup.py file)

After that, just follow the onscreen prompts issued by the command line.

**Comment by J12345654321** [comment]
When it prompts me for a password its not taking anything?

**Comment by jjjchens235** [comment]
I think the text you enter is invisible, just type your pw and then press enter. You'll know it's working correctly when the prompt prints out a hashed value of your password (hashing means your password is converted to a different set of seemingly random characters).

**Comment by J12345654321** [comment]
im still doing something wrong, where do I get the path from?

**Comment by jjjchens235** [comment]
I would google "how to get file path for windows" (or mac, if that's what you're using)

The path you pasted looks strange, I doubt that's correct.

**Comment by J12345654321** [comment]
I originally downloaded the file in ZIP format and extracted the files to a folder.  They never went to the a user file.  Was that wrong?

**Comment by DimaRazb** [comment]
@J12345654321 
1. Install python and all pre-reqs!!!!
2. Run setup.py (should ask login-password.
3. Run BingRewards.py
4. Enjoy.

**Comment by jjjchens235** [comment]
Probably not best practice, but it should be fine.
Assuming the path you listed is where bing-rewards-master folder is saved, then follow these steps. I'm assuming you're in Windows.

1. Open command prompt
2. type _cd C:\bing-rewards-master\_
3. type _python_ setup.py

**Comment by J12345654321** [comment]
I think that did it!  Thank you for the help.  One more quick question, can this run multiple accounts?

**Comment by jjjchens235** [comment]
Glad to hear! And no it cannot.
